easy_rec/python/compat/feature_column/feature_column_v2.py

```diff
@@ -3122,8 +3124,19 @@ def _old_get_dense_tensor_internal(self, sparse_tensors, weight_collections,
           trainable=self.trainable and trainable,
           partitioner=self.partitioner,
           collections=weight_collections)
-    return self._get_dense_tensor_internal_helper(sparse_tensors,
-                                                  embedding_weights)
+    layer_utils.update_attr_to_collection(compat_ops.GraphKeys.RANK_SERVICE_EMBEDDING,
+                                          layer_utils.gen_embedding_attrs(
```

Comment:
add some comments, such as:  for rtp serving purpose

Reply:
Comment added